app/views/early_years_payment/provider/start/claims/check_your_email.html.erb

```diff
@@ -20,7 +20,7 @@
 
     <p class="govuk-body">
       If you do not receive the email, check your spam or junk folder.
-      If you cannot find the email, <%= govuk_link_to "send another link", "" %> or <%= govuk_link_to "enter another email address", @backlink_path %>.
+      If you cannot find the email, <%= govuk_link_to "send another link", claim_path(current_journey_routing_name, "email-address"), method: :patch %> or <%= govuk_link_to "enter another email address", @backlink_path %>.
```

Comment:
is this ok?
There's a couple of issues with this:
* we haven't used method: :patch on a link before, hitting 'back' will prompt for form resubmission
* hardcoded "email-address"